tests/unit/mixins/Registry.ts

```diff
@@ -57,7 +57,7 @@ registerSuite({
 			class IntegrationTest extends TestWithRegistry {
 				render() {
 					return v('div', [
-						w('test', { id: `${Math.random()}` })
+						w<any>('test', { id: `${Math.random()}` })
```

Comment:
@kitsonk I always upgrade the downstream application(s), which will include the examples, as well as `widgets`. We wouldn't want to provide an overload that removed the generic entirely, because as i mentioned it recommended to actually tell the function the typing when referencing a registry item.

I am not recommending using `any`, it was really just a lazy change for the test. It should be the shape that provides the typings for the registry widget, so that you fulfil the property API and also conform to the new children API.